Octokit/Models/Response/Committer.cs

```diff
@@ -1,61 +0,0 @@
-﻿using System;
-using System.Diagnostics;
-using System.Globalization;
-
```

Comment:
Duplicated file (it's also under `Models/Common`) - the new `project.json` file found this twice and it triggered a compilation error.